lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -7,7 +7,22 @@
 const { SyncWaterfallHook } = require("tapable");
 const Template = require("../Template");
 
+const mainTemplateHooksMap = new WeakMap();
+
 class JsonpMainTemplatePlugin {
+	static hooks(mainTemplate) {
```

Comment:
`hooks` -> `getHooks`

Reply:
Alright. I’ll do it in the other PR also